app/app.test.js

```diff
@@ -318,6 +318,22 @@ describe("app", () => {
               expect(msg).toEqual("Bad request: invalid query input");
             });
         });
+        xtest("status: 200 - accepts a topic query and filters values by that topic", () => {
+          return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({ body: { articles } }) => {
+              expect(articles).toHaveLength(11);
+            });
+        });
```

Comment:
Might also be nice to include a test for querying by a topic that does exist, but no associated articles - similar to the comment scenario